libc/include/llvm-libc-types/ACTION.h

```diff
@@ -6,9 +6,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef __LLVM_LIBC_TYPES_ACTION_H__
-#define __LLVM_LIBC_TYPES_ACTION_H__
+#ifndef LLVM_LIBC_TYPES_ACTION_H__
```

Comment:
@michaelrj-google I feel like I should clean up these trailing `__` as well. Thoughts?

Reply:
I haven't yet read the sources of this clang-tidy check, but a comment in a test for it seems to betray that I might need to manually fix up trailing underscores.

https://github.com/llvm-mirror/clang-tools-extra/blob/5c40544fa40bfb85ec888b6a03421b3905e4a4e7/unittests/clang-tidy/LLVMModuleTest.cpp#L60